code/game/machinery/firealarm.dm

```diff
@@ -326,3 +326,109 @@
 	if (!party_overlay)
 		party_overlay = iconstate2appearance('icons/turf/areas.dmi', "party")
 	A.add_overlay(party_overlay)
+
+/obj/machinery/sprinkler
+	name = "foam dispenser"

```

Comment:
This is the only time it's called foam and/or dispenser. Should probably just be called sprinkler for consistency.